base_external_dbsource/__init__.py

```diff
@@ -19,6 +19,4 @@
 #
 ##############################################################################
```

Comment:
Please switch to newer shortened header - https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3